consensus/scp/tests/mock_network/mod.rs

```diff
@@ -314,12 +246,6 @@ impl SCPNetwork {
     }
 }
 
-impl Drop for SCPNetwork {
```

Comment:
I don't think it is necessary to impl Drop 
https://www.reddit.com/r/rust/comments/6cognt/do_i_have_to_manually_implement_drop/

The tests panic with this Drop implementation because the thread handles are not passed around properly... we can try to find and fix the actual bug or just let rust manage cleanup.

Reply:
They currently do not panic, and you do need to implement Drop when you have threads if you want to ensure that when the object that manages those threads goes out of scope the threads have ended. Otherwise they might linger around, which is a bug.